lib/modules/manager/regex/readme.md

```diff
@@ -51,6 +51,8 @@ If you're looking for an online regex testing tool that supports capture groups,
 Be aware that backslashes (`'\'`) of the resulting regex have to still be escaped e.g. `\n\s` --> `\\n\\s`.
 You can use the Code Generator in the sidebar and copy the regex in the generated "Alternative syntax" comment into JSON.
 
+The Regex manager is using [RE2 which does not support backreferences and lookahead assertions](https://github.com/uhop/node-re2#limitations-things-re2-does-not-support). The Regex manager matches are done per-file and not per-line, you should be aware when using the `^` and/or `$` regex assertions.
```

Comment:
```suggestion
The `regex` manager uses RE2 which does not support [backreferences and lookahead assertions](https://github.com/uhop/node-re2#limitations-things-re2-does-not-support).
The `regex` manager matches are done per-file and not per-line, you should be aware when using the `^` and/or `$` regex assertions.
```

Changes:

- Use `monospace` style for the `regex` manager
- Replace `is using` -> `uses`
- Use one sentence per line
- Shorten the link text